feats/feats.yml

```diff
@@ -23,28 +27,37 @@
     As a focus action, you may change between any two forms (including your primary form or any alternate form). You maintain this capability in all of your forms.
     Each form is treated as a completely different character for mechanical purposes - possessing different attributes, feats, perks, flaws, and other defining characteristics. Your alternate form does, however, retain the ability to transform back into your primary form.
     In order to keep track of hit points, you should always record the total damage that your character has suffered. When transforming, your damage remains with you even if your maximum hit points change. For example, Dr. Jekyll has a max HP of 15 and Mr. Hyde has a max HP of 30. During combat, Mr. Hyde suffers 10 damage. When he later transforms back into Dr. Jekyll, the 10 damage remains and is subtracted from his new maximum, leaving the doctor with 5 out of 15 hit points.
-
   special: |
     When selecting feats for your alternate form, you may not select the <strong>Alternate Form</strong> feat.
     You may take this feat multiple times. Each time grants you access to an additional form.
 - !
   name: Area Manipulation (I - V)
   prerequisites:
     tier1:
-      Other:
-      - None
+      all:
+        Other:
+          all:
+          - None
     tier2:
-      Other:
-      - None
+      all:
+        Other:
+          all:
+          - None
```

Comment:
@brianfeister I know we are moving forward, but part of me still thinks there must be a better way. What we have now is nonetheless a solution. Maybe in the future I will come up with something simpler. I have resolved the merge conficts, so that you can merge this when the new application logic is ready.